source/graphql/migrate-hasura.txt

```diff
@@ -0,0 +1,263 @@
+.. _migrate-hasura:
+
+Migrate GraphQL to Hasura
+=========================
+
+.. meta::
+  :description: Learn how to migrate your GraphQL host from Atlas App Services to Hasura.
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. important::
+
+   Always refer to the official documentation of both MongoDB Atlas and `Hasura 
+   <https://hasura.io/docs/latest/databases/mongodb/mongodb-atlas/>`_
+   for the most up-to-date and accurate information. Specific steps may vary
+   depending on the details of your project and the technologies
+   used.
+
+Hasura empowers developers to rapidly build and deploy GraphQL and `REST APIs
+on MongoDB <https://hasura.io/graphql/database/mongodb>`_ and many other data
+sources. By radically cutting down API development times, Hasura enables rapid
+access to data, reduces friction across teams and services, and enables
+enterprises to shorten time to market on data-powered products and features.
+
+Before You Begin
+----------------
+
+If you haven't already, Create an account on Hasura's website `cloud.hasura.io 
+<https://cloud.hasura.io/signup?utm_source=mongodb&utm_medium=blog&utm_campaign=mongodb_developer_portal>`_.
+
+Migrate to Hasura
+-----------------
+
+Migrating your GraphQL API endpoints from MongoDB Atlas App Services to Hasura
+involves a multi-step process that encompasses setting up your environment in
+Hasura, configuring database connections, migrating schemas, and implementing
+authorization and authentication mechanisms. Below is an expanded guide
+detailing each step, with a focus on authorization and role-based access control
+(RBAC) within Hasura. For more information, check out the `Hasura docs 
+<https://hasura.io/docs/latest/databases/mongodb/mongodb-atlas/>`_.
+
+To migrate to Hasura:
+
+#. Create a new project in Hasura
```

Comment:
Something to consider as a fast follow after we roll out these docs - since we're using an ordered list here, should we add the ordered list steps to the headers below? Or - should we make this a high-level procedure where each step is _also_ a procedure? I did a PR like this in one of the other repos during a bug bash or the Atlas Top 250 stuff. You can use different styles to make the numbering look different.

I ask because I started the first procedure and then was confused when it ended at 4 steps, because I was pretty sure the overall procedure above was longer, and then I thought I might lose my place if I'm doing a lot of scrolling around.

Reply:
+1, doesn't need to be a blocker for tomorrow's release but would be good as a follow-up

Reply:
I generally dislike numbered steps in headings - they can be tricky for accessibility.

I'll try doing the high-level procedure approach.